microhapulator/api.py

```diff
@@ -687,24 +687,25 @@ def plot_haplotype_calls(result, outdir, sample=None, plot_marker_name=True, ign
 def get_reads_in_marker_loci(fullref_bam_file, mhindex):
     fullref_bam = pysam.AlignmentFile(fullref_bam_file, "rb")
     reads_to_markers = defaultdict(list)
-    for locus in mhindex.loci.values():
-        start, end = locus.span(chrom=True)
+    for locus, marker in mhindex:
+        start, end = marker.span(chrom=True)
         for read in fullref_bam.fetch(locus.chrom, start, end):
             if not skip_read(read):
-                reads_to_markers[read.query_name].append(locus.id)
+                reads_to_markers[read.query_name].append(marker.id)
     return reads_to_markers
```

Comment:
This is the main part that needed to be changed. Basically I just replaced `locus` with `marker`